### PR TITLE
Svdpowernan

### PR DESCRIPTION
**Pull request by pseudotensor**
Both give nan's.  cuda-memcheck shows no errors.

pytest -s -v tests_open/glm/test_glm_sklearn.py::TestGlmSklearn::test_glm_sklearn_gpu_data0 tests_open/svd/test_tsvd.py::test_tsvd_error_k2

cuda-memcheck pytest -s -v tests_open/glm/test_glm_sklearn.py::TestGlmSklearn::test_glm_sklearn_gpu_data0 tests_open/svd/test_tsvd.py::test_tsvd_error_k2 &> dog7.txt

### CODE REVIEW

File: src/gpu/tsvd/tsvd.cu [hunk 1]
```diff
@@ -380,10 +380,11 @@ void power_tsvd(Matrix<float> &X, double* _Q, double* _w, double* _U, double* _e
 
 	for(int i = 0; i < _param.k; i ++){
 		//Set aside vector of randoms (n x 1)
-		b_k.random(i);
+		b_k.random(_param.random_state + i);
 		float previous_eigenvalue_estimate = FLT_MAX;
 		float eigen_value_estimate = FLT_MAX;
-		while(true){
+		for(int iter=0; iter<_param.n_iter;iter++){
+    		fprintf(stderr,"k=%d/%d iter=%d/%d\n",i,_param.k,iter,_param.n_iter); fflush(stderr);
```
Comment (navdeep-G) [inline src/gpu/tsvd/tsvd.cu]: Might want to wrap this in a verbose call?

File: src/gpu/tsvd/tsvd.cu [hunk 2]
```diff
@@ -392,6 +393,7 @@ void power_tsvd(Matrix<float> &X, double* _Q, double* _w, double* _U, double* _e
 			normalize_vector_cublas(b_k1, context);
 			b_k.copy(b_k1);
 			previous_eigenvalue_estimate = eigen_value_estimate;
+			fprintf(stderr,"%g %g : %g < %g * %g\n",eigen_value_estimate, previous_eigenvalue_estimate, std::abs(eigen_value_estimate - previous_eigenvalue_estimate),_param.tol,std::abs(previous_eigenvalue_estimate)); fflush(stderr);
```
Comment (navdeep-G) [inline src/gpu/tsvd/tsvd.cu]: See previous.

File: tests_open/test_tsvd.py [hunk 3]
```diff
@@ -1,149 +0,0 @@
-import numpy as np
```
Comment (navdeep-G) [inline tests_open/test_tsvd.py]: Deleting this test?

Comment (navdeep-G) [reply tests_open/test_tsvd.py]: Oh, saw the diff up there in svd dir

**Comment by navdeep-G** [comment]
Assuming all relocation of files was done with `git mv`? Also, can I close #417 ?

**Comment by pseudotensor** [comment]
Just for testing and communication purposes...closing as other fixed.

**Pull request closed without merging**